catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -93,20 +102,29 @@ const SearchForm = ({
   }, []);
 
   function updateUrl(form: HTMLFormElement) {
-    const sortOrder = inputValue(form['sortOrder']) || searchParams.sortOrder;
+    const workTypeCheckboxes = nodeListValue(form['workType']) || [];
+    const selectedWorkTypesArray = [...workTypeCheckboxes].filter(
+      selectedWorkType => selectedWorkType.checked
+    );
+
+    const workType =
+      selectedWorkTypesArray.length > 0
+        ? selectedWorkTypesArray.map(workType => workType.value)
+        : null;
+
+    const sortOrder = inputValue(form['sortOrder']);
```

Comment:
As a separate PR I'd like to sit and refine this into _a_ way of getting values from the form to the submission.